requirements.dev.txt

```diff
@@ -11,3 +9,4 @@ pytest-cov
 pytest-subtests
 types-setuptools
 -r docs/sphinx-requirements.txt
+-r requirements.flake8.txt
```

Comment:
```suggestion
-r requirements-flake8.txt
```

Reply:
While I agree that `requirements-???.txt` is more conventional, I will leave this for now so it is in line with the rest of the files, I will maybe make a separate PR to change them all when I can.